storage/pebble.go

```diff
@@ -3,13 +3,44 @@ package storage
 import (
 	"errors"
 	"fmt"
+	"math"
 
 	"github.com/cockroachdb/pebble"
 	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"go.uber.org/multierr"
 
 	storageErrors "github.com/gnolang/tx-indexer/storage/errors"
 )
 
+const (
```

Comment:
Unsure if these should live in pebble.go (ie. specific to the pebbleDB implementation) or be in a separate file (ie. usable by potential other impl's).